config/config.go

```diff
@@ -60,19 +60,19 @@ func LoadConfigVars() (*SyringeConfig, error) {
 		config.CurriculumDir = curriculumDir
 	}
 
+        /*
+                OPTIONAL
+        */
+
 	// +syringeconfig SYRINGE_DOMAIN is used when directing iframe resources to the appropriate place.
 	// Specify the full domain you're using to access the environment.
 	domain := os.Getenv("SYRINGE_DOMAIN")
 	if domain == "" {
-		return nil, errors.New("SYRINGE_DOMAIN is a required variable.")
+		config.Domain = "antidote-local"
```

Comment:
```suggestion
		config.Domain = "localhost"
```

Since the original issue was opened, we've made changes to selfmedicate that make it no longer necessary to use this hostname, so let's default to `localhost`. Otherwise, this change looks good - thank you!

Reply:
Ok, sounds good